### PR TITLE
[MessageBar] aria hidden for message bar icon

### LINKED ISSUE #9051: MessageBar:- In scan mode focus is going to sensory icon and doesn't provide any information while navigate using Up/Down arrow key.
## Actual behavior:
In scan mode focus going to sensory "(i)" icon and doesn't provide any information while navigate using Up/Down arrow key.

## Expected behavior:
In scan mode focus should not go to sensory "(i)" icon while navigate using Up/Down arrow key. 

sensory_icon

## Documentation describing expected behavior
In MessageBar component we are using the "Icon" component for this

issue repro with IE + Jaws Version : 2019.1904.29 ILM

### PR DESCRIPTION
**Pull request by jdhuntington**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #9051
- [X] Include a change request file using `$ npm run change`

#### Description of changes

Hide message bar icon from ARIA tree

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9125)

### CODE REVIEW

File: common/changes/office-ui-fabric-react/messagebar-icon-aria_2019-05-16-22-35.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "[MessageBar] aria hidden for message bar icon",
```
Comment (cliffkoh) [inline common/changes/office-ui-fabric-react/messagebar-icon-aria_2019-05-16-22-35.json]: nit: we usually use this format
```suggestion
      "comment": "MessageBar: aria hidden for message bar icon",
```

File: packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.base.tsx [hunk 2]
```diff
@@ -99,7 +99,7 @@ export class MessageBarBase extends BaseComponent<IMessageBarProps, IMessageBarS
 
   private _getIconSpan(): JSX.Element {
     return (
-      <div className={this._classNames.iconContainer}>
+      <div className={this._classNames.iconContainer} aria-hidden="true">
```
Comment (cliffkoh) [inline packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.base.tsx]: nit: can avoid redundancy 
```suggestion
      <div className={this._classNames.iconContainer} aria-hidden>
```

**Comment by cliffkoh** [comment]
@msft-github-bot hold this pr for 18 hours (to give JD a chance to consider non-blocking feedback)

**Comment by cliffkoh** [comment]
@msft-github-bot delay merge for another 4 hours

**Comment by cliffkoh** [comment]
@msft-github-bot hold this pr for another 4 hours

**Comment by jdhuntington** [comment]
Thanks for the feedback @cliffkoh - good suggestions.

**Pull request merged**